skbio/alignment/tests/test_tabular_msa.py

```diff
@@ -1793,79 +1884,75 @@ def test_index_override(self):
 
 
 class TestAppend(unittest.TestCase):
```

Comment:
I re-wrote/organized/named the tests for `append` and `extend`, the diff here won't be very helpful to reviewers.